src/modules/addInput.js

```diff
@@ -0,0 +1,15 @@
+import Scoresvalue from './fetchInput.js';
+
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+const id = 'hiwi25/';
+const addScore = async (data) => {
+  await fetch(`${api}${id}scores/`, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-type': 'application/json',
+    },
+  }).then((response) => response.json());
+};
+Scoresvalue();
```

Comment:
- [ ] [OPTIONAL] I think you meant to call the `displayScorevalue()` function at line `14` here because calling the `Scoresvalue()` function isn't producing anything. 😉